ui/src/components/role/RoleMember.js

```diff
@@ -109,8 +122,13 @@ export default class RoleMember extends React.Component {
                     >
                         {' '}
                         {review && (
-                            <StyledTagFullNameExpiry>
-                                {' | '}
```

Comment:
Removed this ' | ' on purpose. Not related to my fix but there was a separator accidentally placed at the beginning of the name before the reminder icon (for pending members only). 
Removed it so it will match the way role members look in the active members' list.